br/pkg/backup/client.go

```diff
@@ -859,8 +874,8 @@ func (bc *Client) BackupRange(
 			req.EndKey = progressRange.Incomplete[0].EndKey
 		}
 
-		push := newPushDown(bc.mgr, len(allStores))
-		err = push.pushBackup(ctx, req, progressRange, allStores, bc.checkpointRunner, progressCallBack)
+		push := newPushDown(bc.mgr, len(targetStores))
+		err = push.pushBackup(ctx, req, progressRange, targetStores, bc.checkpointRunner, progressCallBack)
```

Comment:
When the `replicaLabels` is an empty map, will `targetStores` be `nil` here? If so, all the backup procedure would be executed in the fine-grained backup, which is almost unacceptable.

Reply:
Thank you for catching this, updated. PTAL